stdlib/src/builtin/float_literal.mojo

```diff
@@ -315,6 +346,32 @@ struct FloatLiteral(
         return self.__truediv__(rhs).__floor__()
 
     @always_inline("nodebug")
+    fn __mod__(self, rhs: Self) -> Self:
+        """Return the remainder of self divided by rhs.
+
+        Args:
+            rhs: The value to divide on.
+
+        Returns:
+            The remainder of dividing self by rhs.
+        """
+        var quotient: Self = self.__floordiv__(rhs)
+        return self - (quotient * rhs)
```

Comment:
Up to you, but since `__divmod__` is `@always_inline("nodebug")`, you could delegate to it (the compiler should clean up the dead code):
```suggestion
        return self.__divmod__(rhs)[0]
```

Reply:
Good idea, thanks. Done here: https://github.com/msaelices/mojo/commit/7877e00f2e1d50b5e800beb0d4f258802b890e93